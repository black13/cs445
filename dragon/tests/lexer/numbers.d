0
1
012
0123.4
0.1234
123e502134
123e1
123e+4
123e-5
123E-4
125E+6
0.1234e4
0.1231E+6
1234.134E-5
