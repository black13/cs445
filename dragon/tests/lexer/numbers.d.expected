NUM(0)
NUM(1)
NUM(012)
NUM(0123.4)
NUM(0.1234)
NUM(123e502134)
NUM(123e1)
NUM(123e+4)
NUM(123e-5)
NUM(123E-4)
NUM(125E+6)
NUM(0.1234e4)
NUM(0.1231E+6)
NUM(1234.134E-5)
EOF
-- done dumping tokens --
